build.rs

```diff
@@ -141,6 +185,7 @@ fn write_testsuite_tests(
     testsuite: &str,
     strategy: &str,
     pooling: bool,
+    reference_types: bool,
```

Comment:
Could this logic, in addition to issues related to pooling and threads, all get resolved in `tests/all/wast.rs`? The `wast.rs` file already has a lot of logic for dealing with various special cases and ideally this build script is just "go call the `wast.rs` function with all the found tests"

Reply:
done